src/australian-snags.md

```diff
@@ -13,7 +13,7 @@ especially national events. Sometimes served at hardware chains.
 - Onions (4g per sausage)
 - Tomato sauce
```

Comment:
You missed a spot:
```suggestion
- Dead horse
```
garn m8

Reply:
Uhh ackshually, *real* Aussies use 'bottle o' horse'. ;)

Reply:
Yeah, nah, yeah.